File: Homework evaluation/PHP MVC & SYMFONY OVERVIEW - EXERCISES/4/Homework/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{

    private $operator;
    private $leftOperand;
    private $rightOperand;
    // TODO add class fields and properties, getters and setters

    public function getLeftOperand()
    {
        return $this->leftOperand;
    }

    public function getRightOperand()
    {
        return $this->rightOperand;
    }

    public function getOperator()
    {
        return $this->operator;
    }

    public function setLeftOperand($left)
    {
        $this->leftOperand = $left;

        return $this;
    }

    public function setRightOperand($right)
    {
        $this->rightOperand = $right;

        return $this;
    }

    public function setOperator($operator)
    {
        $this->operator = $operator;

        return $this;
    }

    public static function returnResult($operator, $left, $right)
    {
        switch ($operator) {
            case '+' : {
                return $left + $right;
                break;
            }

            case '-' : {
                return $left - $right;
                break;
            }

            case '*' : {
                return $left * $right;
                break;
            }

            case '/' : {
                return $left / $right;
                break;
            }

            case '%' : {
                return $left % $right;
                break;
            }

            case '^' : {
                return $left ** $right;
                break;
            }

            default : {
                return 0;
                break;
            }
        }
    }
}

File: Homework evaluation/JAVA BASIC WEB - EXERCISES - NO PHYSICAL PRESENCE/Homework 3/Homework 3/src/main/java/com/softuni/model/Calculator.java
package com.softuni.model;

public class Calculator {
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return this.leftOperand;
    }

    public double getRightOperand() {
        return this.rightOperand;
    }

    public String getOperator() {
        return this.operator;
    }

    public double calculateResult(){
        double result;

        switch (this.operator){
            case "+":
                result=this.leftOperand+this.rightOperand;
                break;
            case "-":
                result=this.leftOperand-this.rightOperand;
                break;
            case "*":
                result=this.leftOperand*this.rightOperand;
                break;
            case "/":
                result=this.leftOperand/this.rightOperand;
                break;
                default: result=0.0;
                break;
        }
        return result;
    }
}
